scripts/go-get

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -e
+MODULE=$GOGET_MODULE
+VERSION=$GOGET_VERSION
+
+go get -d "${MODULE}@${VERSION}"
+go mod tidy
+go mod verify
+
+if grep -q $MODULE pkg/apis/go.mod; then
```

Comment:
Should we also include `pkg/client/go.mod`? It is used by the terraform provider.